utils/healthcheck.ts

```diff
@@ -128,15 +133,14 @@ export const checkUrlHealth = (url: string): HealthCheck<"Url", true> =>
  * @returns either true or an array of error messages
  */
 export const checkApplicationHealth = (): HealthCheck<ProblemSource, true> =>
-  taskEither
-    .of<ReadonlyArray<HealthProblem<ProblemSource>>, void>(void 0)
-    .chain(_ => checkConfigHealth())
-    .chain(config =>
+  pipe(
+    TE.of(void 0),
+    TE.chain(_ => checkConfigHealth()),
+    TE.chainW(config =>
       // TODO: once we upgrade to fp-ts >= 1.19 we can use Validation to collect all errors, not just the first to happen
```

Comment:
This comment is useless now.

Reply:
Actually no: even if we moved to v2, we didn't implement this feature yet in this project.

Reply:
Right! I correct the assertion above.

Reply:
As soon as we merge https://github.com/pagopa/io-functions-commons/pull/203 we can remove healthcheck.ts at all

Reply:
And so... remove the comment. 😃